swatch-core/src/main/java/org/candlepin/subscriptions/db/model/TallyInstanceViewKey.java

```diff
@@ -64,6 +64,6 @@ public class TallyInstanceViewKey implements Serializable {
   private HardwareMeasurementType measurementType;
 
   @Enumerated(EnumType.STRING)
-  @Column(name = "uom")
+  @Column(name = "metric_id")
   private Measurement.Uom uom;
```

Comment:
I didn't rename the field name here because the same reasoning as in https://github.com/RedHatInsights/rhsm-subscriptions/pull/2415/files#r1299783305.
However, we could certainly rename it. Let me know if you want me to do it.

Reply:
Please add a card to https://issues.redhat.com/browse/SWATCH-1394 for this.